runtime/lua/vim/treesitter/query.lua

```diff
@@ -249,7 +249,32 @@ local predicate_handlers = {
     end
 
     return false
-  end
+  end,
+
+  ["in-plain-string-set?"] = (function()
```

Comment:
The name is a bit confusing. We had suggestions `any-of?` `contained-in?` I guess also `in-string-set?` would be ok.

Reply:
I like `any-of?`, so I'll use that. `contained-in?` might conflict in mental space with Vim highlight syntax `containedin=ALL`, etc.